### DISCUSSION TITLE
Running pretrained weights and model for Blenderbot 2.0

### DISCUSSION THREAD

**Discussion by rvong65** [facebookresearch/ParlAI#4070] [Q&A]
I understand that the pretrained weights are found when running `parlai interactive -mf zoo:blenderbot2/blenderbot2_400M/model --search-server relevant_search_server` but is there any way we can load the pretrained weights into another model and still see the same results? Is it also possible to purely implement Blenderbot2 in Pytorch?

**Comment by klshuster** [comment]
> is there any way we can load the pretrained weights into another model and still see the same results

Yes, you can specify `--init-model zoo:blenderbot2/blenderbot2_400M/model` to initialize a models with the blenderbot2 weights

> Is it also possible to purely implement Blenderbot2 in Pytorch?

I'm not really sure I understand this question; the BB2 model are currently implemented using Pytorch modules.
